spotinst_sdk2/test/managed_instance/aws/test_aws_ma_model.py

```diff
@@ -79,8 +79,7 @@ def runTest(self):
 
 class TestAwsManagedInstanceIntegrationsLoadBalancersConfiguration(AwsManagedInstanceTestCase):
     def runTest(self):
-        lb = LoadBalancer(name="name", arn="arn", type="MULTAI_TARGET_SET", balancer_id="lb-1ee2e3q",
-                          target_set_id="ts-3eq", az_awareness=True, auto_weight=True)
+        lb = LoadBalancer(name="name", arn="arn")
```

Comment:
type is required either CLASSIC or TARGET_GROUP

Reply:
Fixed.